blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DAbstractMetricsRW.java

```diff
@@ -50,6 +48,18 @@ protected DAbstractMetricsRW(LocatorIO locatorIO) {
     protected abstract DAbstractMetricIO getIO( String rollupType, Granularity gran );
 
     /**
+     * Returns the Timer.Context object for timing the Datastax write
+     * @return
```

Comment:
Not relevant. The model for Astyanax is not the same as Datastax. In Datastax, we fire a bunch of Insert statements, and loop over individual ```ResultSetFuture```.

In Astyanax, we prepare a MutationBatch with a bunch of Insert statements, and then we call ```mutationBatch.execute()```. 

The programming model is different.